lib/data/weather/download_all_US_airports_weather.py

```diff
@@ -1,17 +1,35 @@
 import pandas as pd
 import historical_weather_download
 
+# =============================================================================
+# user needs to specify file system locations
+# =============================================================================
+
+#1.  location of airports_list.csv and filtered_airport_codes.csv
+#    example: "C:/Users/ashmui/Documents/GitHub/azure-ai-hackathon"
+
+config_files_loc=input("location of airports_list.csv and filtered_airport_codes.csv, in double quotes: \n")
+
+#2.  desired output location of output files (CSVs of historical by airport, and 1 CSV of all airports appended together)
+#    example: "C:/Users/ashmui/Documents/GitHub/azure-ai-hackathon/for_testing"
+
+output_files_loc=input("desired output location of output files, in double quotes: \n")
+
+
+# =============================================================================
+# start process
+# =============================================================================
 #read in list of airports from the list with latitudes
-df_airports = pd.read_csv("C:/Users/ashmui/Documents/GitHub/azure-ai-hackathon/airports_list.csv"
+df_airports = pd.read_csv(config_files_loc+"/airports_list.csv"
```

Comment:
Instead of reading from local storage. I suggest reading from Azure Datasets. That way, we're sure to be using the same files:
```python
from azureml.core import Workspace, Dataset, workspace
workspace = Workspace.from_config()

# %% Get datasets from ML workspace
# These are created on ML studio
weather = Dataset.get_by_name(workspace, name='all_weather')
covid = Dataset.get_by_name(workspace, name='us_state_covid')
airlines = Dataset.get_by_name(workspace, name='airline_list')
airports = Dataset.get_by_name(workspace, name='airport_list')
flights = Dataset.get_by_name(workspace, name='all_flights')

weather = weather.to_pandas_dataframe()
covid = covid.to_pandas_dataframe()
airlines = airlines.to_pandas_dataframe()
airports = airports.to_pandas_dataframe()
flights = flights.to_pandas_dataframe()
```
But you can also create a Dataset from the default datastore on the fly:
```python
workspace = Workspace.from_config()
datastore = workspace.get_default_datastore()

# create a dataset referencing the cloud location
weather = Dataset.Tabular.from_delimited_files(path = [(datastore, ('airport_list.csv'))])
weather = weather.to_pandas_dataframe()
```

Reply:
Performed this change in a new commit:
df03979dc5778d890a6df0d54916f342eadfc301